Google.Api.Generator.Tests/ProtoTest.cs

```diff
@@ -83,29 +84,25 @@ public void ProtocExecution()
             Run("ProtoTest.proto", "testing");
         }
 
-        [Fact]
-        public void ProtoTests()
+        private void ProtoTestSingle(string testProtoName)
```

Comment:
Rather than have separate tests, use a Theory here with InlineData?

Reply:
I'm using separate tests as it makes it much easier to execute just a single test case; which is very useful during this development.
I'll add a TODO in the next PR to consider changing it later.